packages/opentelemetry-instrumentation-llamaindex/tests/test_query_pipeline.py

```diff
@@ -44,18 +45,52 @@ def test_query_pipeline(exporter):
     spans = exporter.get_finished_spans()
 
     assert {
-        "llama_index_query_pipeline.workflow",
-        "retrieve.task",
-        "synthesize.task",
+        # "QueryPipeline.llama_index.workflow",
```

Comment:
the comment is because the change you pushed to llamaindex? I'd rather write this test as if your PR is already in, and disable this entire test temporarily with a comment as to why (+link to your PR)

Reply:
Wow, PR has been already merged yesterday, so it should be available soon hopefully. I have adjusted to test.